- No bash specific functions, use standard POSIX shell
- Use a much simpler construct to determine the existance of dbus-send.

--- data/tools/enlightenment_remote.orig	2010-01-11 19:31:40.000000000 +0100
+++ data/tools/enlightenment_remote	2010-01-11 19:35:45.000000000 +0100
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 #
 # See the function show_help_new_tool in order to know more about this tool
 
@@ -206,11 +206,7 @@
 #       RETURNS:  just finishes and exit
 #===============================================================================
 
-for dir in $( echo $PATH | tr ':' ' ' )
-do
-   test -x ${dir}/dbus-send && dbus_send_found=yes
-done
-if [[ "$dbus_send_found" != "yes" ]] ; then
+if ! type dbus-send > /dev/null 2>&1; then
    echo "E: dbus-send command not found, please install it first"
    exit 1
 fi
